### PR TITLE
Add Sentry

### PR DESCRIPTION
**Pull request by Silberschleier**
Sentry is a web based error tracker: https://sentry.io and https://github.com/getsentry/sentry

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -8371,6 +8371,15 @@
       "icon": "SDL Tridion.png",
       "website": "http://www.sdl.com/products/tridion"
     },
+    "Sentry": {
+      "cats": [
+        13
+      ],
+      "html": "<script[^>]*>\\s*Raven\\.config\\('[^']*', {\\s+release: '([0-9\\.]+)'\\;version:\\1",
```
Comment (jvoisin) [inline src/apps.json]: You can use this instead:
```json
"script": {
  "Raven.config": "",
  "ravenOptions.whitelistUrls": ""
}
```

File: src/apps.json [hunk 2]
```diff
@@ -8371,6 +8371,15 @@
...
+      "icon": "Sentry.svg",
+      "implies": "Django",
```
Comment (jvoisin) [inline src/apps.json]: This is wrong: one can use sentry without using Django.

Comment (Silberschleier) [reply src/apps.json]: Sentry itself is build with Django. So shouldn't this be added?

File: src/apps.json (hunk 1 above)
Comment (Silberschleier) [reply src/apps.json]: If I see this right, this wouldn't get the version. So should I just add this additionally?

Comment (jvoisin) [reply src/apps.json]: Sure :)

File: src/apps.json (hunk 2 above)
Comment (jvoisin) [reply src/apps.json]: Google Analytics' backend is likely written in Java, but it doesn't mean that the site using it is written in Java too.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**